### ISSUE TITLE
Remove/reduce 3rd party warning messages to screen 

### ISSUE THREAD

**Issue by Anders-Markvardsen** [fitbenchmarking/fitbenchmarking#673]
**To reproduce**
On Windows (and if I heard correctly @wathen also observed on other environment you tried).

Install FitBenchmarking and with default settings simply from command line type and run `fitbenchmarking`. Output observed include:

```
##################################
 Running data from: MGH17.dat 7/9
##################################
    Starting value: 1/2
        Software: SCIPY
            Minimizer: Nelder-Mead
            Minimizer: Powell
            Minimizer: CG
                Jacobian: scipy 2-point
<string>:1: RuntimeWarning: overflow encountered in exp
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:454: RuntimeWarning: invalid value encountered in subtract
  df = fun(x) - f0
<string>:1: RuntimeWarning: overflow encountered in exp
<string>:1: RuntimeWarning: overflow encountered in exp
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:454: RuntimeWarning: invalid value encountered in subtract
  df = fun(x) - f0
<string>:1: RuntimeWarning: overflow encountered in exp
<string>:1: RuntimeWarning: overflow encountered in exp
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:454: RuntimeWarning: invalid value encountered in subtract
  df = fun(x) - f0
<string>:1: RuntimeWarning: overflow encountered in exp
<string>:1: RuntimeWarning: overflow encountered in exp
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:454: RuntimeWarning: invalid value encountered in subtract
  df = fun(x) - f0
<string>:1: RuntimeWarning: overflow encountered in exp
<string>:1: RuntimeWarning: overflow encountered in exp
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:454: RuntimeWarning: invalid value encountered in subtract
  df = fun(x) - f0
<string>:1: RuntimeWarning: overflow encountered in exp
            Minimizer: BFGS
                Jacobian: scipy 2-point
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:454: RuntimeWarning: invalid value encountered in subtract
  df = fun(x) - f0
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:454: RuntimeWarning: invalid value encountered in subtract
  df = fun(x) - f0
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:454: RuntimeWarning: invalid value encountered in subtract
  df = fun(x) - f0
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:454: RuntimeWarning: invalid value encountered in subtract
  df = fun(x) - f0
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:454: RuntimeWarning: invalid value encountered in subtract
  df = fun(x) - f0
            Minimizer: Newton-CG
                Jacobian: scipy 2-point
            Minimizer: L-BFGS-B
                Jacobian: scipy 2-point
            Minimizer: TNC
                Jacobian: scipy 2-point
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:454: RuntimeWarning: invalid value encountered in subtract
  df = fun(x) - f0
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:361: RuntimeWarning: invalid value encountered in less
  if np.any((x0 < lb) | (x0 > ub)):
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:361: RuntimeWarning: invalid value encountered in greater
  if np.any((x0 < lb) | (x0 > ub)):
c:\python37\lib\site-packages\scipy\optimize\_numdiff.py:102: RuntimeWarning: invalid value encountered in greater_equal
  sign_x0 = (x0 >= 0).astype(float) * 2 - 1
        Software: SCIPY_LS
            Minimizer: lm-scipy-no-jac
<string>:1: RuntimeWarning: overflow encountered in exp
<string>:1: RuntimeWarning: invalid value encountered in add
<string>:1: RuntimeWarning: overflow encountered in multiply
            Minimizer: lm-scipy
                Jacobian: scipy 2-point
            Minimizer: trf
                Jacobian: scipy 2-point
            Minimizer: dogbox
                Jacobian: scipy 2-point
    Starting value: 2/2
        Software: SCIPY
```  

In this issue remove these scipy\optimize from the standard console, but please ensure there are capture elsewhere in some logs.

**Comment by wathen** [comment]
Currently the output grabber collects `sys.stdout`, this needs to be extended to `sys.stderr` as well to remove the runtime warnings

**Comment by Anders-Markvardsen** [comment]
Fixed in PR #697 it seems. This issue was not automated closed then. fixes or closes keyword not used in that PR
